src/cascadia/TerminalSettingsEditor/Launch.cpp

```diff
@@ -8,17 +8,43 @@
 
 using namespace winrt;
 using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Controls;
+using namespace winrt::Windows::Foundation;
 using namespace winrt::Microsoft::Terminal::Settings::Model;
 
 namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 {
     Launch::Launch()
     {
         InitializeComponent();
+
+        // find default profile index and initialize dropdown to that Profile
+        const auto profiles{ MainPage::Settings().AllProfiles() };
+        const auto defaultProfileGuid{ GlobalSettings().DefaultProfile() };
+        for (uint32_t i = 0; i < profiles.Size(); ++i)
+        {
+            auto profileGuid{ profiles.GetAt(i).Guid() };
+            if (profileGuid == defaultProfileGuid)
+            {
+                DefaultProfile().SelectedIndex(i);
+            }
+        }

```

Comment:
Now that `DefaultProfile` is bound to a collection of `Profiles`, you can actually initialize `SelectedItem` to be the Default Profile! e.g. something like `DefaultProfile().SelectedItem(FindProfile(DefaultProfileGuid))`

Reply:
yes, please stop maintaining a bunch of separate lists of stuff

Reply:
can you just bind `SelectedItem`? You don't even need to INITIALIZE it to anything. You can just bind it to an IInspectable that _is the right profile_, yes? You would do a bind on the setter and on the getter, and then you'd replace the Changed handler with the Setter ...

Reply:
Had to bind it to a `CurrentDefaultProfile` variable. What throws everything off is the `FindProfile` call because `DefaultProfile` doesn't actually return a Profile. This looks so much cleaner though.